modules/programs/khal-calendar-accounts.nix

```diff
@@ -5,8 +5,8 @@ with lib;
 {
   options.khal = {
     type = mkOption {
-      type = types.nullOr (types.enum [ "calendar" "discover" ]);
-      default = null;
+      type = types.enum [ "calendar" "discover" ];
+      default = "calendar";
```

Comment:
isn't discover more robust ? aka supports more calendars/includes the "calendar" scenario

Reply:
Not really with `discover` it searches for subfolder structure in `local.path` (as in this [example](https://lostpackets.de/khal/configure.html#exemplary-discover-usage)). Without a subfolder structure no calendar is found (I haven't tested if the calendar entries in the root folder `local.path` would even be registered).
One user had it set to `discover` at first and had problems because of that: https://github.com/nix-community/home-manager/issues/4675#issuecomment-1968611460

So I think "calendar" is the more common scenario, at least that's what I am using for two completely different CalDav calendar.